### ISSUE TITLE
PF_Role should become Role (at least for now)

### ISSUE THREAD

**Issue by RieksJ** [AmpersandTarski/Ampersand#935]
For the NavBar feature, the `SystemContext` introduces the concept `PF_Role` as an equivalent of the SIAM concept `Role`. Since **it prevents me from doing work in many projects** I have marked this issue as critical. Here are some of the problems

1. there is no backward compatibility at all. When people use SIAMv3 or any version before that, the definition of the relation `sessionAllowedRoles :: SESSION * PF_Role` ensures that wherever this relation is used, an ambiguity arises that has to be addressed explicitly. Note that this issue is not going away after the current development stuff is moved to the master - it will only provide agony, and will raise issues because of SIAM incompatibilities.

2. for those that intermittently use the PF in development, master, or any other mode, this not only means that they have to keep track of it in their own sources, but also that the file `SystemContext.adl` itself produces ambiguity errors.

So regardless of whether or not it is a good idea to distinguish between `PF_Role` as a role for the prototype framework and `Role` for any other purpose - but in particular for SIAM, I don't see a reason to have this, in particular as the `EQUALS` statement (see #851) is not implemented yet, and since the statement by @sjcjoosten that you can create 'EQUALS' Concepts using `CLASSIFY` has proven to be incorrect - it doesn't work.

@Michiel-s : can you please rename `PF_Role` into `Role`? Note that there do not seem to be problems by having it an Object - in fact it currently already _is_ an Object, so you can add labels etc. to it anyway.

**Comment by Michiel-s** [comment]
I thought we’d discussed this and decided to add `CLASSIFY Role IS PF_Role` to solve the SIAM integration.

Conceptually we are talking about different concepts, so we shouldn’t use the same name. Since we haven’t implemented namespaces yet, the only way to distinguish the two concepts is by given them different names. In this case I chose to prefix with `PF_` (as abbr for Prototype Framework). This way we can easily replace this when namespaces are implemented.

**Comment by Michiel-s** [comment]
Relates to #850

**Comment by RieksJ** [comment]
We've discussed this, yes. And we have made an effort to do it as you stated. 

However, my current practice is that I cannot do my work any more. Also, I expect the &-'students' to run into trouble as well - basically because they use the (new) SIAM source and the old or new PF and the old or new &-master release. 

Having to choose between the ability of doing our project work and preparing for stuff that isn't there yet (i.e. using the `PF_` prefix for the prototype framework) I have taken the liberty to use the single concept of `Role` in an attempt to get going again.

In the meantime, let's continue the discussion. My current position is to use `Role` until there is a way of doing it like you suggest that has been proven to be (sufficiently) backward compatible and allows us to coninue our work. This may mean that we have to wait until #850 is solved or equality between concepts works.
